libraries/plugins/witness/witness.cpp

```diff
@@ -235,6 +241,12 @@ block_production_condition::block_production_condition_enum witness_plugin::bloc
       case block_production_condition::exception_producing_block:
          elog( "exception producing block" );
          break;
+      case block_production_condition::shutdown:
+         ilog( "shutdown producing block" );
+         return result;
+      default:
+         elog( "unknown condition while producing block" );
```

Comment:
Please add the actual ``result`` to the log message.

Reply:
ok sure ! Thanks ! 
Do you mean `default` case only ? Or both cases with `shutdown` also ?

Thanks !

Reply:
Only the default case. In the other cases the message is sufficient to identify the case.